### DISCUSSION TITLE
roadmap about streampark in 2024

### DISCUSSION THREAD

**Discussion by ihadoop** [apache/streampark#3544] [General]
hi，i want to know about the roadmap  in 2024

**Comment by caicancai** [comment]
@ihadoop Thanks for the reminder, the StreamPark community currently only has a 2.2.0 roadmap, not a roadmap for 2024, but your suggestion is probably a good one, the community will try to figure out if it needs to specify a roadmap for 2024

**Reply by ihadoop** [reply]
ok
